app/channels/layer_channel.rb

```diff
@@ -1,17 +1,51 @@
+# TODO: Improve logging
+
 class LayerChannel < ApplicationCable::Channel
-  # TODO: store diffs broadcasted by owner client in db
-  # TODO: receive diffs from participant and store them in db
+  TANGIBLE_DIFF = 'tangible'.freeze
+  REMOVE_DIFF = 'remove'.freeze
+  TRANSLATE_DIFF = 'translate'.freeze
+  FETCH_EXISTING_SIGNAL = 'fetch-existing'.freeze
+
+  # set_callback :after_subscribe, :subscribed
```

Comment:
This comment is no longer relevant right?

Reply:
Deleted